src/audio.c

```diff
@@ -113,7 +113,7 @@ struct autx {
 		RE_ATOMIC bool run;   /**< Audio transmit thread running   */
 	} thr;
 
-	mtx_t lock;
+	mtx_t mtx;
```

Comment:
this diff is already quite large ...

please make the renaming lock->mtx a separate PR